docs/about.md

```diff
@@ -8,14 +8,84 @@ If you are a developer, consult the [Developer Getting Started Guide](https://se
+
+By providing the PROPRIETARY DATA, PROVIDER agrees to the following:
+
+- PROVIDER has the authority to provide the PROPRIETARY DATA to the RECIPIENT, that the information provided constitutes PROPRIETARY DATA, and consents to the information provided being labeled and treated as PROPRIETARY DATA. If PROVIDER wishes to provide non-proprietary data, please contact BTODataTools@ee.doe.gov.
+
+- RECIPIENT may host, display, and manage such PROPRIETARY DATA for the PROVIDER and the Government to access, analyze, and manage through the SEED Platform.
+
+- RECIPIENT may use, view, or duplicate the PROPRIETARY DATA and may share the PROPRIETARY DATA with support services contractors within the scope of their contracts. RECIPIENT may share the PROPRIETARY DATA to other DOE National Laboratories participating in the SEED Platform project, under the restriction that (i) the PROPRIETARY DATA be retained in confidence and not be further disclosed, (ii) the PROPRIETARY DATA will be destroyed when the DOE National Laboratory’s participation in the SEED Platform has ended.
+
+Download a copy of the [data privacy policy](resources/SEED Data Privacy Terms.pdf).
+
+Please [contact us](contact.md) if you have any questions about this privacy policy. Any data uploaded to National Lab hosted SEED instances or shared with NREL/LBNL is proprietary and not to be shared.
+
+#### Data Security
+
+The National Renewable Energy Laboratory and Lawrence Berkeley National Laboratory (LBNL) have implemented a number of software and security controls to ensure the security of data stored in the SEED Platform&trade; software.
+
+SEED Platform&trade; is secured behind the Amazon Web Services (AWS) firewall, and other security measures are in place as well. For more information about specific security protocols, please contact us using the information found on the [contact page](contact.md).
+
+## Additional Hosting-Related Questions
+
+### Can the web interface of a self-hosted SEED instance be accessed via a local network, or is internet access required? Can access via local internet be disabled?
+
+Access via public internet would be a function of the self-hosted instance's network configuration. If SEED is hosted on Amazon Web Services, for example, the host can deny access based on CIDR blocks (IP address ranges) in the Amazon Web Services Console. If the application is hosted on user-owned infrastructure, a local firewall can prevent external access. Note that internal access in both cases would still be available (i.e., the web interface would be accessible).
+
+### What is the estimated cost to implement and maintain the SEED Platform as part of a benchmarking system?
+
+There are three hosting options:
+
+1. Self-hosting on user-owned hardware
+2. Self-hosting on the cloud (e.g. Amazon, Google Compute)
+3. Paying a hosting provider. Several companies offer SEED as a hosted solution:
```

Comment:
Need to add 
* DOE/National Lab-Hosted Instance

Add it to the end of the list.

Reply:
done!